docs/data/material/migration/migration-v4/migrating-from-jss.md

```diff
@@ -0,0 +1,567 @@
+# Migrating from JSS
+
+<p class="description">This guide explains how to migrate from JSS to Emotion when updating from Material UI v4 to v5.</p>
+
+## Material UI v5 migration
+
+1. [Getting started](/material-ui/migration/migration-v4/)
+2. [Breaking changes part one: style and theme](/material-ui/migration/v5-style-changes/)
+3. [Breaking changes part two: components](/material-ui/migration/v5-component-changes/)
+4. Migrating from JSS <— _you are here_
+5. [Troubleshooting](/material-ui/migration/troubleshooting/)
+
+## Migrating from JSS to Emotion
+
+One of the biggest changes in v5 is the swapping of JSS for [Emotion](https://emotion.sh/docs/introduction) (or styled-components as an alternative).
+
+Note that you may continue to use JSS after migrating to v5, and when you're ready to move over to the new styling engine, you can refactor your components progressively.
+
+This document reviews all the steps necessary to migrate away from JSS.
```

Comment:
It's important here to describe which steps from here are a must (e.g. you need to migrate the `styleOverrides` as they are processed with emotion), but you don't need to do the rest if you don't want to (or at least you can do it progressively).

Considering this, maybe we should move the section "Migrate theme styleOverrides to Emotion" to https://deploy-preview-32740--material-ui.netlify.app/material-ui/migration/v5-style-changes/ and than mark this page as (optional) in both the sidebar and the title of the page. I think this will help people better understand what is mandatory and what is optional step in the migration.